### ISSUE TITLE
StickyHeaders are not very smooth

### ISSUE THREAD

**Issue by osdnk** [Flipkart/recyclerlistview#309]
While scrolling I see that headers ale dropping frames on sticking. Can't it be done more smoothly using `Animated` and native drivers?

**Comment by ananyachandra14** [comment]
Yes we'll eventually be using native drivers but for our V0 release our effort was to keep it cross-platform and compatible with web as well. Regardless it shouldn't be dropping frames. Can you share an expo project so I can have a look at it?
